web-bluetooth/index.html

```diff
@@ -33,6 +34,8 @@ <h3>Combining multiple operations</h3>
 
 <h3>Scanning</h3>
 <p><a href="scan.html">Scan for Advertisements</a> - Basic scan for advertisements.</p>
```

Comment:
Let's try to reduce confusion developers may have between scanning for all AD packets, vs advertisements for a single permitted device.  I'm tempted to even break these out into different section headers.  
a) move the watchAdvertisements up to the 'Beginner' section? (with watch + connect to the multiple operations section?)
b) add more clarifying text after "scan for advertisements" that it scans for all advertisements out there. (and I'd place it at the end of the list because we plan to ship it later and it is more powerful).

Reply:
Fixed in https://github.com/GoogleChrome/samples/pull/676/commits/05c36024e0b30c87b30adc224beea2e0328c18c6